app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -124,31 +124,31 @@ def default_authentication_type
   end
 
   def scan_job_create(entity)
-    check_policy_prevent(:request_containerimage_scan, entity, :raw_scan_job_create, entity)
+    check_policy_prevent(:request_containerimage_scan, entity, :raw_scan_job_create)
   end
 
-  def raw_scan_job_create(entity)
+  def raw_scan_job_create(class_name, id)
```

Comment:
Oh my I didn't see the serialization in the log only because I caused the job not not run anymore :open_mouth: I will dig dipper